LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -152,57 +519,234 @@
 		<skos:definition xml:lang="en">insurance policy providing the mortgage indemnity guarantee</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;Purchase">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanPurpose"/>
-		<rdfs:label xml:lang="en">Purchase</rdfs:label>
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
+		<rdfs:label>mortgage loan purpose</rdfs:label>
+		<skos:definition>the purpose for which mortgage loan proceeds will be used, such as real property purchase, dwelling construction, or loan refinancing</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;Remortgage">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanPurpose"/>
-		<rdfs:label xml:lang="en">Remortgage</rdfs:label>
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-BusinessOrCommercial">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>business or commercial</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-DwellingConstruction">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>dwelling construction</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-HomeImprovement">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>home improvement</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-LoanRefinancing">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>loan refinancing</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-MortgageModification">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>mortgage modification</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-RealPropertyPurchase">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>real property purchase</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;NMLSR-ID">
+		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Law/LegalCapacity/LicenseIdentifier"/>
+		<rdfs:label>Nationwide Mortgage Licensing System and Registry Identifier</rdfs:label>
+		<skos:definition>the number permanently assigned by the Nationwide Mortgage Licensing System and Registry (NMLS) for each company, branch, and individual that maintains a single account on NMLS.</skos:definition>
```

Comment:
what is NMLS? 
Should there not be links to the Scheme and Registry?

Reply:
Yes, for another day and in a US-specific subontology.